Index: Custom/Kernel/Modules/AgentTicketZoom.pm
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Custom/Kernel/Modules/AgentTicketZoom.pm	(date 1512640699000)
+++ Custom/Kernel/Modules/AgentTicketZoom.pm	(date 1512641674000)
@@ -1287,6 +1287,24 @@
                     Data => $ZoomMenuItems{$Item},
                 );
             }
+	        ##########
+	        #
+	        # Custom change
+	        # QuickResponsibleChange
+	        #
+	        ##########
+	        elsif ( $ZoomMenuItems{$Item}->{Block} && $ZoomMenuItems{$Item}->{Block} eq 'DocumentMenuItemQuickResponsibleChangeForm' ) {
+		        $LayoutObject->Block(
+		            Name => 'DocumentMenuItemQuickResponsibleChangeForm',
+		            Data => $ZoomMenuItems{$Item},
+		        );
+	        }
+	        ##########
+	        #
+	        # End of custom change
+	        # QuickResponsibleChange
+	        #
+	        ##########
 	        else {
                 $LayoutObject->Block(
                     Name => 'TicketMenu',
Index: Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewMedium.tt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewMedium.tt	(date 1512640699000)
+++ Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewMedium.tt	(date 1512641156000)
@@ -252,3 +252,25 @@
     </form>
 </li>
 [% RenderBlockEnd("DocumentMenuItemMoveForm") %]
+##########
+#
+# Custom change
+# QuickResponsibleChange
+#
+##########
+[% RenderBlockStart("DocumentMenuItemQuickResponsibleChangeForm") %]
+<li class="[% Data.Class | html %]">
+    <form title="[% Translate("QuickResponsibleChange ticket") | html %]" action="[% Env("CGIHandle") | html %]" method="post">
+        <input type="hidden" name="Action" value="AgentTicketQuickResponsibleChange"/>
+        <input type="hidden" name="TicketID" value="[% Data.TicketID | html %]"/>
+        <label for="QuickResponsibleChange" class="InvisibleText">[% Translate("QuickResponsibleChange") | html %]:</label>
+        [% Data.NewResponsibleStrg %]
+    </form>
+</li>
+[% RenderBlockEnd("DocumentMenuItemQuickResponsibleChangeForm") %]
+##########
+#
+# End of custom change
+# QuickResponsibleChange
+#
+##########
Index: Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewPreview.tt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewPreview.tt	(date 1512640699000)
+++ Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewPreview.tt	(date 1512641156000)
@@ -280,3 +280,25 @@
     </form>
 </li>
 [% RenderBlockEnd("DocumentMenuItemMoveForm") %]
+##########
+#
+# Custom change
+# QuickResponsibleChange
+#
+##########
+[% RenderBlockStart("DocumentMenuItemQuickResponsibleChangeForm") %]
+<li class="[% Data.Class | html %]">
+    <form title="[% Translate("QuickResponsibleChange ticket") | html %]" action="[% Env("CGIHandle") | html %]" method="post">
+        <input type="hidden" name="Action" value="AgentTicketQuickResponsibleChange"/>
+        <input type="hidden" name="TicketID" value="[% Data.TicketID | html %]"/>
+        <label for="QuickResponsibleChange" class="InvisibleText">[% Translate("QuickResponsibleChange") | html %]:</label>
+        [% Data.NewResponsibleStrg %]
+    </form>
+</li>
+[% RenderBlockEnd("DocumentMenuItemQuickResponsibleChangeForm") %]
+##########
+#
+# End of custom change
+# QuickResponsibleChange
+#
+##########
Index: Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewSmall.tt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewSmall.tt	(date 1512640699000)
+++ Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketOverviewSmall.tt	(date 1512641155000)
@@ -381,3 +381,25 @@
 [% RenderBlockEnd("DocumentColumnFilterFormElement") %]
 </form>
 [% RenderBlockEnd("DocumentColumnFilterForm") %]
+##########
+#
+# Custom change
+# QuickResponsibleChange
+#
+##########
+[% RenderBlockStart("DocumentMenuItemQuickResponsibleChangeForm") %]
+<li class="[% Data.Class | html %]">
+    <form title="[% Translate("QuickResponsibleChange ticket") | html %]" action="[% Env("CGIHandle") | html %]" method="post">
+        <input type="hidden" name="Action" value="AgentTicketQuickResponsibleChange"/>
+        <input type="hidden" name="TicketID" value="[% Data.TicketID | html %]"/>
+        <label for="QuickResponsibleChange" class="InvisibleText">[% Translate("QuickResponsibleChange") | html %]:</label>
+        [% Data.NewResponsibleStrg %]
+    </form>
+</li>
+[% RenderBlockEnd("DocumentMenuItemQuickResponsibleChangeForm") %]
+##########
+#
+# End of custom change
+# QuickResponsibleChange
+#
+##########
Index: Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketZoom.tt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketZoom.tt	(date 1512640699000)
+++ Custom/Kernel/Output/HTML/Templates/Standard/AgentTicketZoom.tt	(date 1512641156000)
@@ -191,6 +191,28 @@
                                 <a href="[% Env("Baselink") %]Action=AgentTicketMove;TicketID=[% Data.TicketID | uri %];" class="AsPopup PopupType_TicketAction" title="[% Translate("Change Queue") | html %]">[% Translate("Queue") | html %]</a>
                             </li>
 [% RenderBlockEnd("MoveForm") %]
+##########
+#
+# Custom change
+# QuickResponsibleChange
+#
+##########
+[% RenderBlockStart("DocumentMenuItemQuickResponsibleChangeForm") %]
+<li class="[% Data.Class | html %]">
+    <form title="[% Translate("QuickResponsibleChange ticket") | html %]" action="[% Env("CGIHandle") | html %]" method="post">
+        <input type="hidden" name="Action" value="AgentTicketQuickResponsibleChange"/>
+        <input type="hidden" name="TicketID" value="[% Data.TicketID | html %]"/>
+        <label for="QuickResponsibleChange" class="InvisibleText">[% Translate("QuickResponsibleChange") | html %]:</label>
+        [% Data.NewResponsibleStrg %]
+    </form>
+</li>
+[% RenderBlockEnd("DocumentMenuItemQuickResponsibleChangeForm") %]
+##########
+#
+# End of custom change
+# QuickResponsibleChange
+#
+##########
 [% RenderBlockStart("TicketMenuExternalLink") %]
                             <li>
                                 <a href="[% Data.Link | Interpolate %]" class="[% Data.Class | html %]" [% Data.LinkParam %] title="[% Translate(Data.Description) | html %]" target="[% Data.Target %]">[% Translate(Data.Name) | html %]</a>
